README.md

```diff
@@ -1,210 +1,16 @@
-# sysl-catalog
 
```

Comment:
This seems to have been overwritten by mistake, think you can remove this file from the commit?